Content.Client/Storage/ItemCounterSystem.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Content.Shared.Storage;
+using Content.Shared.Storage.ItemCounter;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+
+namespace Content.Client.Storage
+{
+    [UsedImplicitly]
+    public class ItemCounterSystem : SharedItemCounterSystem
```

Comment:
Does the ItemCounterSystem work with only the shared definition? Or does it need these empty systems here to register?